### PR TITLE
Josh'a auto adventures

### PR DESCRIPTION
**Pull request by JoshuaGoldberg**
potential autos just dropped, also some fixes to other autos

### CODE REVIEW

**Review by Sid220 — APPROVED** [review]

File: src/main/java/frc/robot/commands/OnBridgeUntilMovement.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import frc.robot.util.MotionHandler;
+import frc.robot.util.MotionHandler.MotionMode;
+
+public class OnBridgeUntilMovement extends SequentialCommandGroup {
+  double robotSpeed = 0;
+  double backSpeed = 0;
+
+  public OnBridgeUntilMovement(boolean gridside) {
+    robotSpeed = 2 * (gridside ? 1 : -1) * (DriverStation.getAlliance() == Alliance.Red ? -1 : 1);
+    backSpeed = 0.4 * (gridside ? -1 : 1) * (DriverStation.getAlliance() == Alliance.Red ? -1 : 1);
```
Comment (tervay) [inline src/main/java/frc/robot/commands/OnBridgeUntilMovement.java:22]: Can  we simplify this to something like
```
if (condition1 || condition 2) { robotSpeed = ...; } else { robotSpeed = -...; }
```
and the same thing for back speed?

Will be a bit easier to debug.

**Review by tervay — CHANGES_REQUESTED** [review]
just simplify some bridge speed stuff

File: src/main/java/frc/robot/commands/OnBridgeUntilMovement.java [hunk 2]
```diff
@@ -21,6 +20,14 @@ public OnBridgeUntilMovement(boolean gridside) {
     robotSpeed = 2 * (gridside ? 1 : -1) * (DriverStation.getAlliance() == Alliance.Red ? -1 : 1);
     backSpeed = 0.4 * (gridside ? -1 : 1) * (DriverStation.getAlliance() == Alliance.Red ? -1 : 1);
 
+    if((gridside == true && DriverStation.getAlliance() == Alliance.Blue) || (gridside == false && DriverStation.getAlliance() == Alliance.Red)){
```
Comment (tervay) [inline src/main/java/frc/robot/commands/OnBridgeUntilMovement.java:23]: You can simplify this:

```
(gridside && DriverStation.getAlliance() == Alliance.Blue) || (!gridside && DriverStation.getAlliance() == Alliance.Red)
```

you don't need to compare booleans to `true` or `false`.

File: src/main/java/frc/robot/commands/OnBridgeUntilMovement.java [hunk 3]
```diff
@@ -21,6 +20,14 @@ public OnBridgeUntilMovement(boolean gridside) {
     robotSpeed = 2 * (gridside ? 1 : -1) * (DriverStation.getAlliance() == Alliance.Red ? -1 : 1);
```
Comment (tervay) [inline src/main/java/frc/robot/commands/OnBridgeUntilMovement.java:20]: delete these now

**Review by tervay — CHANGES_REQUESTED** [review]

**Review by tervay — APPROVED** [review]

**Pull request merged**